torch/csrc/jit/tensorexpr/expr.cpp

```diff
@@ -134,6 +146,18 @@ Expr trunc(const Expr& v) {
   return Intrinsics::make(kTrunc, v);
 }
 
+Expr sigmoid(const Expr& v) {
+  return Intrinsics::make(kSigmoid, v);
+}
+
+Expr relu(const Expr& v) {
```

Comment:
`relu` isn't an intrinsic, it's `${0} < 0 ? 0.f : ${0}`  (or, `max(${0}, 0.0`, since we have `max` in our DSL).